pyproject.toml

```diff
@@ -7,6 +7,43 @@ force_single_line = true
 line_length = 89
 profile = "black"
 
+[tool.ruff]
```

Comment:
tool.isort should also become tool.ruff.isort

See [here](https://docs.astral.sh/ruff/settings/#isort-case-sensitive)